src/test/java/frc/utils/TestGyro.java

```diff
@@ -0,0 +1,35 @@
+package frc.utils;
```

Comment:
clever class. but we may not need it if we avoid a dependency on a hardware Gyro in the first place.